vendor.conf

```diff
@@ -57,7 +57,8 @@ github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc374
 github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
 github.com/coreos/etcd v3.2.1
 github.com/coreos/go-semver v0.2.0
-github.com/ugorji/go f1f1a805ed361a0e078bb537e4ea78cd37dcf065
+# fix for go vet (https://github.com/kolyshkin/ugorji-go/commit/1cf431c13dec46596)
```

Comment:
we should create a tracking issue for this (ie to get rid of the fork)

Reply:
Sure, https://github.com/moby/moby/issues/37804